src/components/Attachments/AttachmentView/index.tsx

```diff
@@ -95,16 +92,20 @@ function AttachmentView({
     isWorkspaceAvatar,
     maybeIcon,
     fallbackSource,
-    transaction,
+    transactionID = '-1',
```

Comment:
There is a conversation about the usage of `-1` as default here: https://expensify.slack.com/archives/C01GTK53T8Q/p1729014019344209?thread_ts=1728329641.957179&cid=C01GTK53T8Q

In the future I understand we want undefined or null (not sure which one), so I don't know if we should be using `null` or `undefined` or `-1` here.

Reply:
We will continue with `-1` for now and migrate later.